typescript/src/cloudflare/index.ts

```diff
@@ -0,0 +1,59 @@
+import {z, ZodRawShape} from 'zod';
+import {ToolCallback} from '@modelcontextprotocol/sdk/server/mcp.js';
+import {McpServer} from '@modelcontextprotocol/sdk/server/mcp.js';
+import {registerPaidTool} from '../modelcontextprotocol/register-paid-tool';
+import type {PaidToolOptions} from '../modelcontextprotocol/register-paid-tool';
+
+// @ts-ignore: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'.
```

Comment:
switched to `index.ts` instead of `.mts` since that was causing issues when importing this in a different file